pkg/command/dependencies.go

```diff
@@ -1,16 +1,75 @@
 package command
 
 import (
+	"fmt"
+	"strings"
+
 	"github.com/Autodesk/shore/pkg/backend"
+	"github.com/Autodesk/shore/pkg/backend/spinnaker"
+	"github.com/Autodesk/shore/pkg/config"
 	"github.com/Autodesk/shore/pkg/project"
 	"github.com/Autodesk/shore/pkg/renderer"
+	"github.com/Autodesk/shore/pkg/renderer/jsonnet"
 	"github.com/sirupsen/logrus"
 )
 
+// Renderer Enum
+const (
+	// JSONNET - Jsonnet Renderer
+	JSONNET string = "jsonnet"
+)
+
+// Backend Enum
+const (
+	// SPINNAKER - Spinnaker Backend
+	SPINNAKER string = "spinnaker"
+)
+
 // Dependencies - Shared dependencies all controller MAY require
 type Dependencies struct {
-	Renderer renderer.Renderer
-	Backend  backend.Backend
-	Logger   logrus.FieldLogger
-	Project  *project.Project
+	Renderer        renderer.Renderer
+	Backend         backend.Backend
+	Logger          logrus.FieldLogger
+	Project         *project.Project
+	ShoreConfig     config.ShoreConfig
+	ShoreConfigOpts config.ShoreConfigOpts
```

Comment:
Aside from tests, where are these fields used?

Reply:
Currently, they're not being used anywhere.

The intent is to have them there early so when render/save/execute/test-remote commands start making use of the shore config the information will already be there.

For instance, when the render command is ran it will have the shore config loaded in the dependencies as well as the chosen profile - and so it will be able to use the correct config based on the profile.